packages/pong-engine/src/state.ts

```diff
@@ -95,35 +84,23 @@ export const reducer = (
     case 'paddleMoveRight':
       return {
         ...state,
-        paddle: {
-          ...paddle,
-          slide: PADDLE_SLIDE_SPEED,
-        },
+        paddle: movePaddleRight(paddle),
```

Comment:
```
  const { ball, paddle, score } = state;
```
instead of
```
  const ball = state.ball;
  const paddle = state.paddle;
  const score = state.score;
```
?

Reply:
Yes, thank you!